experimenter/experimenter/features/manifests/nimbus_fml_loader.py

```diff
@@ -24,16 +25,19 @@ def __init__(self, application: str, channel: str):
     @classmethod
     @lru_cache
     def create_loader(cls, application: str, channel: str):
+        """Application and channel should stay the same for each iteration of the
```

Comment:
This should be a summary in the imperitive mood. This reads like a comment, not a docstring